demo/lint.html

```diff
@@ -165,7 +165,7 @@
     mode: "css",
     gutters: ["CodeMirror-lint-markers"],
     lint: {
-      disableLintOnChange: true
+      lintOnChange: false
```

Comment:
OK, I'll fix that. Although this is meant to present the use of performLint in the following lines which performs lint every 5 seconds. Do you still think I should remove it?